outputs/aws.go

```diff
@@ -101,6 +103,7 @@ func (c *Client) SendMessage(falcopayload types.FalcoPayload) {
 
 	resp, err := svc.SendMessage(input)
 	if err != nil {
+		c.CountMetric("outputs", 1, []string{"output:" + c.OutputType, "status:error"})

```

Comment:
`OutputType` should be `strings.ToLower`ised, right?

Reply:
Made this change in a few places so the `output` tag is consistently lower-case.